controllers/openshift_controller.go

```diff
@@ -735,3 +743,63 @@ func (r *KataConfigOpenShiftReconciler) SetupWithManager(mgr ctrl.Manager) error
 		For(&kataconfigurationv1.KataConfig{}).
 		Complete(r)
 }
+
```

Comment:
this is exactly what we have on the release-4.6 branch, right?

Reply:
yup.